src/main/java/com/commercetools/paymenttoorderprocessor/jobs/actions/OrderCreator.java

```diff
@@ -74,8 +111,8 @@ private String encrypt(String value) {
             cipher.init(Cipher.ENCRYPT_MODE, ks);
             byte[] encrypted = cipher.doFinal(value.getBytes());
             return Base64.encodeBase64String(encrypted);
-        } catch (Exception ex) {
-            ex.printStackTrace();
+        } catch (NoSuchAlgorithmException | NoSuchPaddingException | InvalidKeyException | IllegalBlockSizeException | BadPaddingException e) {
```

Comment:
1. why we catch only this list of exceptions? isn't it better to localise all potential encryption issues in one place if we already created `try..catch` block?
2. `SecretKeySpec ks = new SecretKeySpec(keyData, ENCRYPTIONALGORITHM);` throws _InvalidKeyException_ if `createorder.encryptionkey` is longer than 128 bits (16 charactes). Why it is restricted to so short key?